### PR TITLE
SPHINX - verticapy.core.vDataFrame._aggregate

### PR DESCRIPTION
**Pull request by oualib**
Docstring for verticapy.core.vDataFrame._aggregate:

Missing docstring for:
 - def describe (vdf & vdc)

Main reviewer: @mail4umar 

@tconsigny please double check **if you get the chance**.

### CODE REVIEW

**Comment by oualib** [comment]
@afard should we reformat the docstring as well? Does black do it automatically?

File: verticapy/core/vdataframe/_aggregate.py [hunk 1]
```diff
@@ -1425,6 +1934,57 @@ def median(
...
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+        
+        Now, let's calculate the median for specific columns.
+
+        .. code-block:: python
+
+            data.median(
+                columns = ["x", "y", "z"],,
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:1957]: you can remove the last extra comma. 

columns = ["x", "y", "z"],**,**

File: verticapy/core/vdataframe/_aggregate.py [hunk 2]
```diff
@@ -1425,6 +1934,57 @@ def median(
...
+            :suppress:
+
+            import verticapy as vp
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+            result = data.median(
+                columns = ["x", "y", "z"],,
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:1971]: Here as well. You can remove the last extra comma. 

columns = ["x", "y", "z"],**,**

File: verticapy/core/vdataframe/_aggregate.py
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:1]: - the function "describe" has been skipped for both vDataFrame and vDataColumn.

- other functions skipped: groupby, product, quantile, duplicated 

- for quite a few functions (aad, any, all, avg, count...), see also consists only of aggregate.

File: verticapy/core/vdataframe/_aggregate.py [hunk 3]
```diff
@@ -1217,6 +1269,56 @@ def aad(
...
+            html_file = open("figures/core_vDataFrame_vDFAgg_aad_table.html", "w")
+            html_file.write(result._repr_html_())
+            html_file.close()
+
+        .. raw:: html
+            :file: SPHINX_DIRECTORY/figures/core_vDataFrame_vDFAgg_aad_table.html
+
+        .. note:: All the calculations are pushed to the database.
+
+        .. hint:: For more precise control, please refer to the `aggregate` method.
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:1317]: we can instead use a reference for the module: :py:mod:`verticapy.vDataColumn.aggregate`

File: verticapy/core/vdataframe/_aggregate.py [hunk 4]
```diff
@@ -1631,6 +2433,55 @@ def sum(
...
+
+            import verticapy as vp
+
+            data = vp.vDataFrame({
+                "x": [1, 2, 4, 9, 10, 15, 20, 22],
+                "y": [1, 2, 1, 2, 1, 1, 2, 1],
+                "z": [10, 12, 2, 1, 9, 8, 1, 3],
+            })
+        
+        Now, let's calculate the maximum for specific columns.
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:2451]: maximum = sum

File: verticapy/core/vdataframe/_aggregate.py [hunk 5]
```diff
@@ -100,16 +100,15 @@ def aggregate(
...
             |    **sem**: standard error of the mean
             |    **skewness**: skewness
             |    **sum**: sum
             |    **std**: standard deviation
-            |    **topk**: kth most occurent element
-                                    (ex: top1 for the mode)
+            |    **topk**: kth most occurent element (ex: top1 for the mode)
             |    **topk_percent**: kth most occurent element density
             |    **unique**: cardinality (count distinct)
             |    **var**: variance
```
Comment (mail4umar) [inline verticapy/core/vdataframe/_aggregate.py:114]: for the line below, plz update the spacing. the word version should not come under the line of "|". Instead "v" of version should align with "O" of others on the top line.

**Review by mail4umar — CHANGES_REQUESTED** [review]

File: verticapy/core/vdataframe/_aggregate.py (hunk 3 above)
Comment (oualib) [reply verticapy/core/vdataframe/_aggregate.py:1317]: They can click on the see also below

File: verticapy/core/vdataframe/_aggregate.py (hunk 4 above)
Comment (oualib) [reply verticapy/core/vdataframe/_aggregate.py:2451]: Corrected but please double check, I've noticed some other mistakes.

File: verticapy/core/vdataframe/_aggregate.py (hunk 5 above)
Comment (oualib) [reply verticapy/core/vdataframe/_aggregate.py:114]: Done but please double check again.

**Comment by tconsigny** [comment]
Just glanced through a lot of the expanded function descriptions. They look good to me.

**Pull request merged**